util/verible-format.sh

```diff
@@ -27,9 +32,12 @@ fi
 git add -u
 
 # get all system verilog files and pipe through style formatter
-find hw/{ip,vendor,top_earlgrey} -type f -name "*.sv" -o -name "*.svh" | \
+find . -type f -name "*.sv" -o -name "*.svh" |                           \
     xargs -n 1 -P $NUM_PROCS verible-verilog-format                      \
-    --inplace
+    $VERIBLE_ARGS
+
+
+verible-verilog-format
```

Comment:
Could you please explain why this script runs `verible-verilog-format` once more?

Reply:
that is.... a copy paste error, thanks for catching :)

Reply:
fixed